_includes/homepage-hero-band.html

```diff
@@ -1,11 +1,16 @@
 <div class="grid-wrapper homepage-hero-band">
-  <div class="grid__item width-12-12">
-    <div class="home-hero">
+  <div class="grid__item width-6-12 width-12-12-m homepage-hero-band-lcontent">
       <img src="{{site.baseurl}}/assets/images/quarkus_logo_horizontal_rgb_reverse.svg" class="project-logo" title="Quarkus">
       <h2>Supersonic Subatomic Java</h2>
-      <p>A Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards</p>
-      <a href="{{site.baseurl}}/get-started/" class="button-cta secondary">Get Started</a>
-      <a href="#" class="scroll-down" address="true"></a>
-    </div>
+      <p>A Kubernetes Native Java stack tailored for GraalVM & OpenJDK HotSpot, crafted from the best of breed Java libraries and standards</p>
```

Comment:
This switches GraalVM to be before OpenJDK, which I know in the past we didn't want.

Has the view changed? @n1hility @emmanuelbernard

Reply:
graalvm in front of openjdk should not happen - I would even flip it around further and just go for writing "A Kubernetes Native Java stack crafted from the best of breed Java libraries and standards tailored for OpenJDK and GraalVM"

Reply:
Yes the upstream site has the right order

Reply:
Meaning OpenJDK first

Reply:
I've updated the text so that it reads "A Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards."